pkg/kubectl/cmd/create_service.go

```diff
@@ -83,6 +82,10 @@ func NewCmdCreateServiceClusterIP(f cmdutil.Factory, cmdOut io.Writer) *cobra.Co
 	return cmd
 }
 
+func errUnsupportedGenerator(cmd *cobra.Command, generatorName string) error {
```

Comment:
@mengqiy what do you think? Move to `kubectl/util`?
I have not done this yet because the function is used only within the package.